Configurations/10-main.conf

```diff
@@ -17,7 +17,7 @@ sub vc_win64a_info {
                                 asflags   => "/c /Cp /Cx",
                                 asoutflag => "/Fo" };
         } else {
-            $die->("NASM not found - please read INSTALL and NOTES.WIN for further details\n");
+            $die->("NASM not found - make sure it's installed or configure with 'no-asm'\n");
```

Comment:
I'd like to argue against advertising no-asm option. At least not like that, as if it's kind of equivalent alternative to installing nasm. no-asm should rather be viewed as debug option than production.

Reply:
Ok.  What would you suggest, then?  I can simply shorten this string to "NASM not found - make sure it's installed"

Reply:
Well, original message was just as good. In the essence it doesn't matter what it says(*), because people won't actually read it. Like in the case in question. Configure script did fail, didn't it? But it didn't actually prevent user from attempting to run nmake... Of course one can say that referred NOTES.WIN wasn't clear enough. But then it might be appropriate to make that message clearer.

(*) as long as it doesn't say things we don't want to say, like suggesting that no-asm is as viable production option as any other.